### ISSUE TITLE
TCanvas wrapper using Skia?

### ISSUE THREAD

**Issue by Alexey-T** [skia4delphi/skia4delphi#120]
It is possible to have such wrapper? I mean lib must work with Skia and export the TCanvas descendant class so my app, with heavy TCanvas.TextOut, Line, Rectangle, FillRect, Circle - may work.

Then I want to help with Lazarus support.

**Comment by Dave-Novo** [comment]
I agree. Having a VCL TCanvas wrapper is quite helpful. Our company would be willing to pay to support these development efforts. Please contact david.novo@denovosoftware.com if one of the maintainers is interested.
